src/main/scala/devices/debug/Debug.scala

```diff
@@ -86,17 +86,22 @@ object DebugAbstractCommandType extends scala.Enumeration {
   *  full specification. The implementation which receives this
   *  object can perform more checks on what that implementation
   *  actually supports.
-  *  nComponents : The number of components to support debugging.
-  *  nDMIAddrSize : Size of the Debug Bus Address
-  *  nAbstractDataWords: Number of 32-bit words for Abstract Commands
-  *  nProgamBufferWords: Number of 32-bit words for Program Buffer
-  *  hasBusMaster: Whether or not a bus master should be included
-  *  maxSupportedSBAccess: Maximum transaction size supported by System Bus Access logic.
-  *  supportQuickAccess : Whether or not to support the quick access command.
-  *  supportHartArray : Whether or not to implement the hart array register (if >1 hart).
-  *  hasImplicitEbreak: There is an additional RO program buffer word containing an ebreak
-  *  crossingHasSafeReset: Include "safe" logic in Async Crossings so that only one side needs to be reset.
-  **/
+  *  @param nComponents Number of components to support debugging.
+  *  @param baseAddress Base offest for debugEntry and debugException
+  *  @param nDMIAddrSize Size of the Debug Bus Address
+  *  @param nAbstractDataWords Number of 32-bit words for Abstract Commands
+  *  @param nProgamBufferWords Number of 32-bit words for Program Buffer
+  *  @param hasBusMaster Whether or not a bus master should be included
+  *  @param clockGate todo
```

Comment:
clockGate is used in [Periphery.scala](https://github.com/chipsalliance/rocket-chip/blob/master/src/main/scala/devices/debug/Periphery.scala#L298), where the debug module is instantiated.